.eslintrc.yaml

```diff
@@ -19,10 +19,28 @@ ignorePatterns:
   - "!/src"
   - "!/test"
```

Comment:
Similar to the comment for the other config, thoughts on inverting this? It requires excluding a few more folders, but it reveals some relevant code in `auspice-client` which wasn't being linted (and has errors)

```
ignorePatterns:
  - "node-modules"
  - "/devData"
  - "dist"
  - "/docs/build"
```